pkg/pod/list.go

```diff
@@ -182,3 +207,85 @@ func WaitForAllPodsInNamespaceRunning(
 
 	return true, nil
 }
+
+// WaitForAllPodsInNamespacesHealthy waits until all pods in a list of namespaces that match options
+// are in healthy state.
+// An pod in a healthy state is in running phase and optionally in ready condition.
+//
+// nsNames lists the list of namespaces to monitor. Monitors all namespaces when empty.
+// timeout is the duration to wait for the pods to be healthy
+// includeSucceeded when true, implies that pods in succeeded phase are running.
```

Comment:
```suggestion
// includeSucceeded when true, checks that pods in succeeded phase are running.
```

Reply:
Fixed. Reworded to clarify the intent. includeSucceeded considers succeeded pods as healthy